ICS Nerva Mapping Updates: Psych, Chapel, Journalist, Medbay Storage, Lounge & Library, Maint + Ladders, Pod 1.

Yantarii opened the pull request:
# Major Changes:      (Needs pipes, disposals, wires, etc fixed before going further)
**Chapel** 
Reworks most of Chapel, add a mass-driver for funeral and 'advanced evidence removal', adds a confession booth for the chaplain and crew, coffin storage maintenance additions, adds further maintenance to the right of the chapel, that lead down to Deck 2, by the EVA storage. More Multi-Z railings and such. Aiming to make the chapel a lot better of a place for both RP and also team antag bases, and such, similar with a lot of the other changes I have made. 

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/008d4094-3333-430c-b57f-7d88901a7fe7)


**Journalist Office** - Moves Journalist to the Psycholigist's Office. Keeps the main foyer from the Psych's for the Journalist.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/fb8e120c-bfef-40b8-bd3a-ffd119665c28)


**Psychologist Office** - Moves to Medbay Washroom, opposite the cloning area. Keeps it out the way, and simplifies the washroom. Keeps the bathroom to allow for sneaky antags to break through another way into medbay, but add normal open-space turf now, that lets you see into the abandoned bar.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/2160b60b-ce47-4ef8-8dce-39600c36a263)


**Medbay Storage (Bottom Level)** - Adds washroom to the left side, compacted it. Moves medical items into medbay storage, to be stored. Makes Medical Storage flush will right maintenance wall.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/670ef5a8-8f3f-4d65-9cbf-6054827ef6f2)


**Lounge:** - Moves the Library Section to a more private area with optional curtains, great for your cult runes or plotting of evil schemes. Moves Arcade closer to the railings.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/9f8c5a42-f2d4-49b4-851a-11ae7a1f93f8)


**Head Dormitories:** - Killed and replaced with additional maintenance, which leads up to Deck 4, near the AI. Minor solar adjustments made to make room for it, Lounge area slightly expanded due to the additional free space.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/a2857030-fc1b-4093-9aac-4aaedc5e7251)
**Deck 3 Ladder.**
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/bd3048ef-16eb-4b7d-aaa2-2ce6451b00b0)
**Deck 4 Ladder.**


**Maintenance:** Tool-Storage maintenance expanded to the left, Medbay Storage maintenance compacted slightly. Head-Dorms turned into additional Maintenance. Deck 4 has additional maintenance room for ladders. Coffin Storage expanded slightly. Moves hidden maint room near Tool Storage. Adds a poker-room near the Chapel connected by a confession booth.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/39842404-ed3f-4a5a-b634-4221e6872539)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/6dbd2f34-7cf2-4e97-85ff-fa63748912c1)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/06ab3c1e-d295-4415-a8a8-df3d6075e1a8)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/0d69dc61-603f-4ef8-8964-427486e3666e)


**Escape Pod 1:** Moved Escape Pod one to Deck 4, near the docking port north in order to make room for rework, Nerva-5 adjusted for a new sideways Pod. Kept the exact same otherwise. Airlock adjusted for this pod.

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/40a127be-0f50-4d40-827b-8dda2e1c3110)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/4a042f13-9d12-44fd-b7d3-a3c2f27c6851)


**Spawners:**
Moves some carp spawners slightly further away. (Deck 4)
Adds blob-spawn near Coffin Storage.

**Cameras** - Cameras adjusted for new areas.
Cameras cannot see into Library section now.
Cameras cannot see into Mourge.
Cameras cannot see into the confession booth.

Full Map Screenshots w/ Changes:

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/dba4d315-5e12-4a7b-a2dd-7ec20d046fb7)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/5d8eea51-e23a-4940-bf04-e42456d6a7e4)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/1e61933c-2628-4d30-b494-3e062d11a5ae)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/bace0e2a-5b7d-4370-97f4-44d174cae393)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/bccf135a-5fef-4c9a-b82f-a8fd413b2396)

CPU-Blanc commented:
If it requires further work, _please_ open it as a draft PR and convert when it's ready for review.

That being said, I'm not too sure I'm personally a fan of the escape pod on deck 1. Looks really out of place.

ErrorsWindows commented:
i think the escape pod can work on first deck but maybe over the reinforced black roof because otherwise aesthetically it would appear goofy sticking out the side like metal gear rex radome

ErrorsWindows commented:
if you're looking for an extra tile of space in the medbay extra storage one thing you could remove is the anesthetics closet, where I went on a whole tirade about how [the anesthetic tank is basically griefing material now](https://discord.com/channels/145674110075273216/581657197910556674/1164668830182809685)

CPU-Blanc commented:
> if you're looking for an extra tile of space in the medbay extra storage one thing you could remove is the anesthetics closet, where I went on a whole tirade about how [the anesthetic tank is basically griefing material now](https://discord.com/channels/145674110075273216/581657197910556674/1164668830182809685)

N20 _does_ have its legitimate usages, I've needed to use it before. If bay has changed the effects and made it useless, that's something that'll need addressing/correcting in a separate PR

Yantarii commented:
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/f36615a3-76a5-49c3-b4c4-b6800813042a)

Adjusts original area of Pod 1, keeps it in line with Pod 2 and 3's designs, where it's chambered off, looks less goofy. Adjusted Tool Storage and Pod 1 to be 1 tile more to the right, and slightly lowered the size of tool storage.

Yantarii commented:
https://github.com/UristMcStation/UristMcStation/pull/1434

Since there'sa bunch of changes to the map, I've added some of the tweaks from this PR in this - Wheelchair added to the medbay storage, I've mirrored two holomaps on two z's (Tool Storage to South Bar)

Added current holomap in hangar deck to be located near the stairs. 

![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/62e9cb08-1a3b-4ea6-bafa-1f9a7c7733ae)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/f843521b-b875-4e2c-822b-f7e9c3aadb80)
![image](https://github.com/UristMcStation/UristMcStation/assets/53942081/0dfa4e78-7374-4e29-883c-5064e9fcb03a)

These are mostly around high traffic areas, especially near new player spawns (should they play assistant) - Tool Storage, HoP Line, etc.

CPU-Blanc commented:
Need to take a closer look at this and test properly, but there does seems to be some random holes/odd shapes on the outer hull now on the floor tiles. Mainly deck 1+2

Yantarii commented:
Matched the outer hulls to the ones below now, so it should be less spotty, and found a couple areas I missed

Yantarii commented:
tried to update to latest baymerge to fix mapping errors but it seems i've made it worst, it's joever
